drivers/i2s/i2s_ll_stm32.h

```diff
@@ -20,6 +20,7 @@ struct i2s_stm32_cfg {
 	const struct pinctrl_dev_config *pcfg;
 	void (*irq_config)(const struct device *dev);
 	bool master_clk_sel;
+	IF_ENABLED(DT_ANY_INST_HAS_BOOL_STATUS_OKAY(ioswp), (bool ioswp;))
```

Comment:
To keep things simple and not add an extra memory cell, would the below work?
```patch
-	bool master_clk_sel;
+	bool master_clk_sel :1;
+	bool ioswp :1;
```
It would remove the need to test `DT_ANY_INST_HAS_BOOL_STATUS_OKAY(ioswp)` here and there.